sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ArithmeticExpressionSuite.scala

```diff
@@ -157,4 +157,29 @@ class ArithmeticExpressionSuite extends SparkFunSuite with ExpressionEvalHelper
     checkEvaluation(Sqrt(-1), null, EmptyRow)
     checkEvaluation(Sqrt(-1.5), null, EmptyRow)
   }
+
+  test("pmod") {
+    checkEvaluation(Pmod(Literal(7), Literal(3)), 1)
+    checkEvaluation(Pmod(Literal(-7), Literal(3)), 2)
+    checkEvaluation(Pmod(Literal(-7), Literal(3)), 2)
+    checkEvaluation(Pmod(Literal(7.0D), Literal(3.0D)), 1.0D)
+    checkEvaluation(Pmod(Literal(7.2), Literal(4.1)), 3.1000000000000005)
+    checkEvaluation(Pmod(Literal.create(null, IntegerType), Literal.create(3, IntegerType)), null)
+    checkEvaluation(Pmod(Literal.create(3, IntegerType), Literal.create(null, IntegerType)), null)
+    checkEvaluation(
+      Pmod(Literal(Decimal(0.7)), Literal(Decimal(0.2)))
+      , Decimal(0.1))
+    checkEvaluation(
+      Pmod(Literal.create(7.0F, FloatType), Literal.create(3.0F, FloatType)), 1.0F)
+    checkEvaluation(
+      Pmod(Literal.create(7.2, DoubleType), Literal.create(4.1, DoubleType)), 3.1000000000000005)
+    checkEvaluation(
+      Pmod(Literal.create(7.toShort, ShortType), Literal.create(3.toShort, ShortType)), 1.toShort)
+    checkEvaluation(
+      Pmod(Literal.create(7.toByte, ByteType), Literal.create(3.toByte, ByteType)), 1.toByte)
+    checkEvaluation(
+      Pmod(Literal.create(7L, LongType), Literal.create(3L, LongType)), 1L)
+    checkEvaluation(
+      Pmod(Literal.create(null, IntegerType), Literal.create(null, IntegerType)), null)
```

Comment:
We don't need to create the `Literal` manually, just write the value like other tests did.